connect/runtime/src/test/java/org/apache/kafka/connect/runtime/standalone/StandaloneHerderTest.java

```diff
@@ -887,19 +783,14 @@ public void testPutConnectorConfig() throws Exception {
 
         assertEquals("bar", capturedConfig.getValue().get("foo"));
         herder.connectorConfig(CONNECTOR_NAME, connectorConfigCb);
```

Comment:
Can we also make sure that there aren't any errors reported with the `connectorConfigCb`?

The easy way to do this would be to just add this line to the end of the test:
```java
verifyNoMoreInteractions(connectorConfigCb);
```

It'd probably be better to do away with mocked callbacks and switch to using a `FutureCallback` that we invoke `get` on (like we do with many other tests), but it's up to you if you'd like to implement that or not.

Reply:
Fixed and committed.



> It'd probably be better to do away with mocked callbacks and switch to using a `FutureCallback` that we invoke `get` on (like we do with many other tests), but it's up to you if you'd like to implement that or not.

I think its better to solve this as another PR/issue. This PR has already been ongoing for long enough and this just appears to be a nice improvement for me.